app/models/navigation.js

```diff
@@ -7,8 +7,8 @@ export default EmberObject.extend({
 	fetchAll(host, wikiId, language) {
 		const url = buildUrl({
 			host,
-			path: `/api/v1/design-system/wikis/${wikiId}/${language}/`,
-			wiki: 'www'
+			/* eslint-disable max-len */
```

Comment:
instead of disabling eslint here
what do you think about
`/wikia.php?controller=DesignSystemApi&method=getAllElements\
&product=wikis&id=${wikiId}&lang=${language}`

Reply:
IMO with disabling eslint the path is more readable and it's at least 2 bytes less code ;)

Reply:
1. maybe :)
2. nope, https://babeljs.io/repl/#?babili=false&browsers=&build=&builtIns=false&code_lz=G4QwTgBCELwQBgGwPaIgHQFADtkAcBTeAbiA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=true&lineWrap=true&presets=es2015%2Ces2016%2Ces2017%2Cstage-0%2Cstage-1%2Cstage-2%2Cstage-3%2Ces2015-loose%2Cenv&prettier=false&targets=&version=6.26.0&envVersion=1.6.2 :)